common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4476,6 +4484,13 @@
     ],
     "sqlState" : "42K06"
   },
+  "STATE_STORE_SCHEMA_FILE_THRESHOLD_EXCEEDED" : {
+    "message" : [
+      "The number of state schema files <numStateSchemaFiles> exceeds the maximum number of state schema files allowed for this query, <maxStateSchemaFiles>.",
```

Comment:
Lets update this and also note the `colFamilyName` for which the threshold breach was seen ?